api_helper.py

```diff
@@ -0,0 +1,27 @@
+import requests
+import logging
+
+class APIFetch:
+    def __init__(self) -> None:
+        pass
+    def fetch_data(self,url,country:None):
+        if country is not None:
+            print("Data of Country and Univerities:")
```

Comment:
this print is not required to add here, because wht if introduce new API which uses this function, it will print this statement for that too